ddtrace/tracer.py

```diff
@@ -176,15 +176,16 @@ def _default_span_processors_factory(
     if single_span_sampling_rules:
         span_processors.append(SpanSamplingProcessor(single_span_sampling_rules))
 
-    span_processors.append(
+    # These need to run after all the other processors
+    deferred_processors = [
```

Comment:
Are we planning to add to the deferred processors list? If not, I think we should avoid adding the span aggregator to a list. The indirection makes this code a bit harder to read.

Reply:
It's hard to predict what will come in the future. I think it is conceivable that there might be processors that need to run _after_ any other registered processors, so the list gives us a bit of "future-proofness"